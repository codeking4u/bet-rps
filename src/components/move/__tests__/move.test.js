import renderer from "react-test-renderer";

import Move from "../move.component";

test("matches snapshot", () => {
  const mockBet = { bet: "ROCK", onClick: jest.fn() };
  const tree = renderer.create(<Move {...mockBet} />).toJSON();
  expect(tree).toMatchSnapshot();
});
