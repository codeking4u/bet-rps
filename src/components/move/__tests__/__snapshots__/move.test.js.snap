// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="betOption rock  "
  onClick={[Function]}
>
  <div>
    ROCK
     
  </div>
</div>
`;
